src/rt/argv_windows.d

```diff
@@ -0,0 +1,253 @@
+module rt.argv_windows;
+
+version (Windows):
+
+import core.stdc.stdlib /+: malloc+/;
+import core.stdc.wchar_ /+: wcslen+/;
+import core.sys.windows.winnt /+: LPWSTR, LPCWSTR+/;
+
+
+/++
+Splits a command line into argc/argv lists, using the VC7 parsing rules.
+This functions interface mimics the `CommandLineToArgvW` api.
+If function fails, returns NULL.
+If function suceeds, call `LocalFree(HLOCAL)` on return pointer when done.
+NOTE Implementation-wise, once every few years it would be a good idea to
+compare this code with the .NET Runtime's `SegmentCommandLine` method,
+which is in `src/coreclr/src/utilcode/util.cpp`.
+Date: Feb 4, 2020
++/
+LPWSTR* commandLineToArgv(LPCWSTR lpCmdLine, int* pNumArgs) nothrow @nogc
+{
+    /+
+    The MIT License (MIT)
```

Comment:
The runtime library is boost licensed in general, I'm not sure we want code with other licenses in here.